### PR TITLE
Street2geek update readme

### PR DESCRIPTION
**Pull request by street2geek**

### CODE REVIEW

File: package-lock.json
Comment (Shigoto-dev19) [inline package-lock.json:1]: Why is the `package-lock.json` changed?

File: README.md [hunk 1]
```diff
@@ -1,6 +1,73 @@
...
+
+The code master generates 4 secret digits in a set sequence. Digits can be between 1-9 and the digits must all be different. 
+Then, each turn, the code breaker tries to guess the code master's digits, who then gives the number of matches.  
+If the matching digits are in their right positions, they are "hits", if they in different positions, they are "blows". 
+
+Example:
+
+Code master private solution: 4 2 7 1 <br />
+Code breaker's public solution: 1 2 3 4 <br />
+Answer: 1 hit and 2 blows. (The hit is "2" in the second position, the blows are "4" and "1".) <br />
```
Comment (Shigoto-dev19) [inline README.md:15]: Actually 2 is a hit and 1 is blow.

File: package-lock.json
Comment (street2geek) [reply package-lock.json:1]: @Shigoto-dev19 The name changed from `name: "package-name"` to `"name": "mina-mastermind"` after did an `npm install`

File: README.md (hunk 1 above)
Comment (street2geek) [reply README.md:15]: Its saying that there are 1 hit and 2 blows in total... With 2 being a hit whilst 4 and 1 being blows. 

I think thats is correct right?

Comment (Shigoto-dev19) [reply README.md:15]: I see, there's some lexical confusion!
- The digit `2` in the example is in the same position and equal and hence it is a hit
- The digit `1` in the example exists in the solution but it's not in the correct position hence it is a blow

Comment (street2geek) [reply README.md:15]: (The hit is "2" in the second position, the blows are "4" and "1")

Comment (Shigoto-dev19) [reply README.md:15]: correct, I overlooked the digit `4` :+1:

**Pull request merged**